### PR TITLE
Create the new Appearance screen

### PR DESCRIPTION
**Pull request by anikiki**
Task/Issue URL: https://app.asana.com/0/0/1204670443433650/f

### Description
New Appearance screen.

### Steps to test this PR

- [ ] Install from this branch.
- [ ] Go to `Settings`, scroll down and tap on the `Appearance` item.
- [ ] Notice the new `Appearance` screen opens and has the same functionality as the old section in `Settings`.
 
### UI changes
| Before  | After |
| ------ | ----- |
|old_appearance_section|appearance_screen|

### CODE REVIEW

**Comment by anikiki** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #3226** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3226" target="_blank">Graphite</a> 
    * **PR #3230** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3230" target="_blank">Graphite</a> 
      * **PR #3246** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3246" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/duckduckgo/Android/3246?utm_source=stack-comment).

**Review by malmstein — APPROVED** [review]
Looks good! Just a nit

File: app/src/main/java/com/duckduckgo/app/appearance/AppearanceViewModel.kt [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+    fun viewState(): StateFlow<ViewState> {
+        return viewState
+    }
+
+    fun commands(): Flow<Command> {
+        return command.receiveAsFlow()
+    }
+
+    fun start() {
```
[the remark this answers is not in the document]
Comment (anikiki) [reply app/src/main/java/com/duckduckgo/app/appearance/AppearanceViewModel.kt]: Updated 👍

File: app/src/main/java/com/duckduckgo/app/appearance/AppearanceViewModel.kt [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+            )
+        }
+    }
+
+    fun userRequestedToChangeTheme() {
+        viewModelScope.launch { command.send(Command.LaunchThemeSettings(viewState.value.theme)) }
+        pixel.fire(AppPixelName.SETTINGS_THEME_OPENED)
+    }
+
+    fun userRequestedToChangeIcon() {
```
[the remark this answers is not in the document]
Comment (anikiki) [reply app/src/main/java/com/duckduckgo/app/appearance/AppearanceViewModel.kt:91]: Added the pixel and updated the test as well.

**Pull request merged**